src/main/java/com/cdancy/bitbucket/rest/BitbucketApi.java

```diff
@@ -20,23 +20,24 @@
 import java.io.Closeable;
 
 import com.cdancy.bitbucket.rest.features.AdminApi;
-import com.cdancy.bitbucket.rest.features.BuildStatusApi;
 import com.cdancy.bitbucket.rest.features.BranchApi;
+import com.cdancy.bitbucket.rest.features.BuildStatusApi;
```

Comment:
:warning: Wrong lexicographical order for 'com.cdancy.bitbucket.rest.features.BuildStatusApi' import. Should be before 'java.io.Closeable'. (#QR82:24#)-CHECKSTYLE_CustomImportOrderCheck